fatbin/fatbin_test.go

```diff
@@ -53,11 +61,6 @@ func TestFatbinNonExist(t *testing.T) {
 }
 
 func TestSniff(t *testing.T) {
-	switch runtime.GOOS {
```

Comment:
How does `Stat` change whether we can run on non-{linux,darwin}?

Reply:
It doesn't. It's an unrelated change. The check was obsoleted by 71f2ad7